proto/raze/remote/protobuf-2.8.2.BUILD

```diff
@@ -0,0 +1,90 @@
+"""
+cargo-raze crate build file.
+
+DO NOT EDIT! Replaced on runs of cargo-raze
+"""
+package(default_visibility = [
+  # Public for visibility by "@raze__crate__version//" targets.
+  #
+  # Prefer access through "//proto/raze", which limits external
+  # visibility to explicit Cargo.toml dependencies.
+  "//visibility:public",
+])
+
+licenses([
+  "notice", # "MIT"
+])
+
+load(
+    "@io_bazel_rules_rust//rust:rust.bzl",
+    "rust_library",
+    "rust_binary",
+    "rust_test",
+)
+
+rust_binary(
+    name = "protobuf_build_script",
+    srcs = glob(["**/*.rs"]),
+    crate_root = "build.rs",
+    edition = "2015",
+    deps = [
+    ],
+    rustc_flags = [
+        "--cap-lints=allow",
+    ],
+    crate_features = [
+      "bytes",
+      "with-bytes",
+    ],
+    data = glob(["*"]),
+    version = "2.8.2",
+    visibility = ["//visibility:private"],
+)
+
+genrule(
+    name = "protobuf_build_script_executor",
+    srcs = glob(["*", "**/*.rs"]),
+    outs = ["protobuf_out_dir_outputs.tar.gz"],
+    tools = [
+      ":protobuf_build_script",
+    ],
+    tags = ["no-sandbox"],
+    cmd = "mkdir -p $$(dirname $@)/protobuf_out_dir_outputs/;"
+        + " (export CARGO_MANIFEST_DIR=\"$$PWD/$$(dirname $(location :Cargo.toml))\";"
+        # TODO(acmcarther): This needs to be revisited as part of the cross compilation story.
+        #                   See also: https://github.com/google/cargo-raze/pull/54
+        + " export TARGET='x86_64-unknown-linux-gnu';"
+        + " export RUST_BACKTRACE=1;"
+        + " export RUSTC=echo;"
+        + " export CARGO_PKG_VERSION=0;"
```

Comment:
I had to add the previous two lines to get this to work. This is definitely a hack, but the `build.rs` script for this target is just trying to generate a VERSION constant which for debugging/logging purposes, so I'm passing in some fake values.

Reply:
It also generates an identifier, such as `VERSION_2_8_2: () = ();`

That is used here:
https://github.com/stepancheg/rust-protobuf/blob/v2.8/protobuf-codegen/src/lib.rs#L160
to make sure protobuf-codegen and protobuf are the same version.

I had looked as using raze to patch the lib.rs file directly, similar to:
```
diff --git a/src/lib.rs b/src/lib.rs
index b487e6e..7a99866 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -114,9 +114,9 @@ mod protobuf {
 
 /// This symbol is in generated `version.rs`, include here for IDE
 #[cfg(never)]
-pub const VERSION: &str = "";
+pub const VERSION: &'static str = "2.8.2";
 /// This symbol is in generated `version.rs`, include here for IDE
 #[cfg(never)]
 #[doc(hidden)]
-pub const VERSION_IDENT: &str = "";
-include!(concat!(env!("OUT_DIR"), "/version.rs"));
+pub const VERSION_IDENT: &'static str = "VERSION_2_8_2";
+pub const VERSION_2_8_2: () = ();
```
but hadn't completely worked through that yet.  The idea was to allow for building without the build.rs file.